components/script/dom/globalscope.rs

```diff
@@ -231,6 +235,23 @@ struct TimerListener {
     context: Trusted<GlobalScope>,
 }
 
+///  A wrapper for the handling of file data received by the ipc router
```

Comment:
nit: Let consistently use one space following `///`(see below where the space is missing, here it looks like there two). `./mach fmt` doesn't catch this I think.